kafka-streams-framework/src/main/java/org/hypertrace/core/kafkastreams/framework/MultiKafkaStreamInstanceApp.java

```diff
@@ -0,0 +1,45 @@
+package org.hypertrace.core.kafkastreams.framework;
+
+import com.typesafe.config.Config;
+import java.util.ArrayList;
+import java.util.List;
+import org.hypertrace.core.serviceframework.PlatformService;
+import org.hypertrace.core.serviceframework.config.ConfigClient;
+
+public abstract class MultiKafkaStreamInstanceApp extends PlatformService {
+
+  static final String SUB_TOPOLOGY_NAMES_CONFIG_KEY = "sub.topology.names";
+
+  private final List<KafkaStreamsApp> appsList = new ArrayList<>();
+
+  public MultiKafkaStreamInstanceApp(ConfigClient configClient) {
+    super(configClient);
+  }
+
+  @Override
+  protected void doInit() {
+    Config appConfig = getAppConfig();
+    List<String> apps = appConfig.getStringList(SUB_TOPOLOGY_NAMES_CONFIG_KEY);
```

Comment:
these are not sub-topologies anymore. these are individual topologies in their respective streams applications.

`child.streams.apps` or `child.apps`?

Reply:
changed to streams.apps